### PR TITLE
fix bug 929095: add frame trust information

### PR DESCRIPTION
**Pull request by willkg**

### CODE REVIEW

**Comment by willkg** [comment]
This can be tested by hand pretty quickly, but there are no unit tests. I'm going to write up another bug to cover adding unit tests for the crashing thread table.

**Comment by willkg** [comment]
Mmm... Actually, I'll do it in this bug.

File: webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html [hunk 1]
```diff
@@ -659,15 +660,14 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
...
-                                    </td>
-                                    {% else %}
-                                    <td>
-                                        {% if frame.file %} {{ frame.file }}:{{ frame.line }} {% endif %}
+                                        {% if frame.source_link %}
+                                            <a href="{{ frame.source_link }}">{{ frame.file }}:{{ frame.line }}</a>
+                                        {% else %}
+                                            {% if frame.file %}{{ frame.file }}:{{ frame.line }}{% endif %}
+                                        {% endif %}
                                     </td>
```
Comment (willkg) [inline webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html]: I got tripped up that there was a `<td>` in each side of the `if` block here so I hoisted that part out and the `if` block only affects the contents now.

**Comment by willkg** [comment]
self-r+

**Comment by asutherland** [comment]
:tada: Yay!  Thanks so much for this! :tada:

**Comment by willkg** [comment]
I'm super sorry I didn't do it earlier. If there are bugs for other changes to that table, let me know--sometimes I don't really understand what bugs are about.

**Pull request merged**